pmsv2/directdebit/params.go

```diff
@@ -0,0 +1,56 @@
+package directdebit
+
+type ChannelCode string
+type ChannelProperties map[string]interface{}
+type Type string
+
+const (
+	// debit card
+	BRI        ChannelCode = "BRI"
+	BCAOneklik ChannelCode = "BCA_ONEKLIK"
+	CIMBNIAGA  ChannelCode = "CIMBNIAGA"
+
+	// bank account
+	BPI       ChannelCode = "BPI"
+	Unionbank ChannelCode = "UBP"
+	RCBC      ChannelCode = "RCBC"
+	BDO       ChannelCode = "BDO"
+	MTB       ChannelCode = "MTB"
+	CHINABANK ChannelCode = "CHINABANK"
+	Mandiri   ChannelCode = "MANDIRI"
+
+	BBL ChannelCode = "BBL"
+	SCB ChannelCode = "SCB"
+	KTB ChannelCode = "KTB"
+	BAY ChannelCode = "BAY"
+
+	// bank redirect
+	BCAKlikpay ChannelCode = "BCA_KLIKPAY"
+)
+
+const (
+	BankAccountType  Type = "BANK_ACCOUNT"
+	DebitCardType    Type = "DEBIT_CARD"
+	BankRedirectType Type = "BANK_REDIRECT"
+)
+
+type CreateMethod struct {
```

Comment:
should only be channel_code and channel_properties

Reply:
updated to be only channel_code and channel_properties